<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>430</width>
    <height>430</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>430</width>
    <height>430</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>430</width>
    <height>430</height>
   </size>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Задание 17</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/source/fon1.png);</string>
  </property>
  <property name="locale">
   <locale language="Russian" country="Russia"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="contextMenuPolicy">
    <enum>Qt::NoContextMenu</enum>
   </property>
   <widget class="QPlainTextEdit" name="TextEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>410</width>
      <height>280</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>410</width>
      <height>280</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>410</width>
      <height>280</height>
     </size>
    </property>
    <property name="cursor" stdset="0">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::NoContextMenu</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 158);
border: 2px solid rgb(255, 255, 158);
background: transparent;
font-size: 18px;</string>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="LineEdit">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>305</y>
      <width>161</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>161</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>161</width>
      <height>30</height>
     </size>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::NoContextMenu</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background: rgb(140, 140, 143);
color: rgb(255, 255, 158);
border: 1px solid black;
border-radius: 4px;
font-size: 14px;</string>
    </property>
    <property name="clearButtonEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>347</y>
      <width>90</width>
      <height>40</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>90</width>
      <height>40</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>90</width>
      <height>40</height>
     </size>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::NoContextMenu</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
color: rgb(255, 255, 158);
font-size: 14px;
border: 1px solid black;
border-radius: 4px;
background: white;
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(238, 238, 255, 255), stop:1 rgba(0, 0, 0, 255));
}

QPushButton:pressed {
color: rgb(255, 255, 158);
font-size: 14px;
background: white;
border: 1px solid black;
border-radius: 4px;
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(212, 209, 180, 0), stop:1 rgba(0, 0, 0, 255));
}</string>
    </property>
    <property name="text">
     <string>Перевести</string>
    </property>
   </widget>
   <widget class="QLabel" name="TextLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>305</y>
      <width>241</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>241</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>241</width>
      <height>30</height>
     </size>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::NoContextMenu</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt; color:#ffff9e;&quot;&gt;Введите кол-во километров&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>430</width>
     <height>33</height>
    </rect>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::NoContextMenu</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QMenuBar {
background: rgb(109, 110, 117);
font-size: 14px;
}

QMenuBar::item {
color: rgb(255, 255, 158);
font-size: 14px;
padding: 4px 12px;
border: 1px solid black;
border-radius: 4px;
background: white;
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(238, 238, 255, 255), stop:1 rgba(0, 0, 0, 255));
 }


QMenuBar::item:pressed {
color: rgb(255, 255, 158);
font-size: 14px;
padding: 3px 6px;
border: 1px solid black;
border-radius: 4px;
background: white;
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(212, 209, 180, 0), stop:1 rgba(0, 0, 0, 255));
}</string>
   </property>
   <widget class="QMenu" name="menu">
    <property name="contextMenuPolicy">
     <enum>Qt::NoContextMenu</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/source/fon.png);
selection-color: rgb(255, 255, 158);
selection-background-color: rgb(109, 110, 117);
color: black;
font-size: 14px;</string>
    </property>
    <property name="locale">
     <locale language="Russian" country="Russia"/>
    </property>
    <property name="title">
     <string>Меню</string>
    </property>
    <widget class="QMenu" name="Language">
     <property name="title">
      <string>Язык</string>
     </property>
     <addaction name="act_rus"/>
     <addaction name="act_eng"/>
    </widget>
    <addaction name="act_author"/>
    <addaction name="act_about"/>
    <addaction name="Language"/>
    <addaction name="separator"/>
    <addaction name="act_esc"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <action name="act_author">
   <property name="text">
    <string>Об авторе</string>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
  </action>
  <action name="act_rus">
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string notr="true">Русский</string>
   </property>
   <property name="menuRole">
    <enum>QAction::ApplicationSpecificRole</enum>
   </property>
  </action>
  <action name="act_eng">
   <property name="text">
    <string notr="true">English</string>
   </property>
  </action>
  <action name="act_about">
   <property name="text">
    <string>О программе</string>
   </property>
  </action>
  <action name="act_esc">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
